tests/corobo_test_case.py

```diff
@@ -0,0 +1,36 @@
+from errbot.backends.test import FullStackTest
+from errbot.plugin_info import PluginInfo
+from errbot.templating import add_plugin_templates_path
+from pathlib import Path
+
+import logging
+
+
+class CoroboTestCase(FullStackTest):
+
+    def setUp(self, plug_filename):
+        super().setUp(loglevel=logging.ERROR,
+                      extra_config={'BACKEND': 'text'})
```

Comment:
text backend? why?

Reply:
Because `test_create_issie_cmd` needed text backend https://github.com/coala/corobo/blob/d697e18d436a33dca07f29e6d26544d3234750f3/tests/labhub_test.py#L165.

If I had not used it. I would have to call `super().setup(...)`at the beginning of every test case.